App/App.pro

```diff
@@ -91,7 +91,9 @@ DISTFILES += \
 RESOURCES += \
     base.qrc \
     json.qrc \
-    translations.qrc
+    translations.qrc \
+    assets.qrc \
+    music.qrc 
```

Comment:
Same reason as mentioned above - `assets.qrc` and `music.qrc` are handled in `assets.pri`